sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateSampleDao.java

```diff
@@ -394,33 +302,16 @@ public Long countBySearch(String querystr) throws IOException {
     criteria.addOrder("asc".equalsIgnoreCase(sortDir.toLowerCase()) ? Order.asc(sortCol) : Order.desc(sortCol));
     @SuppressWarnings("unchecked")
     List<Sample> requestedPage = criteria.list();
-    return fetchSqlStore(requestedPage);
+    return requestedPage;
   }
 
   @Override
   public Collection<Sample> listBySubmissionId(long submissionId) throws IOException {
-    Query query = currentSession().createQuery("from SampleImpl where submissionId like :id");
-    query.setLong("id", submissionId);
-    @SuppressWarnings("unchecked")
-    List<Sample> records = query.list();
-    return fetchSqlStore(records);
-  }
-
-  private Set<DetailedSample> listByParentId(long parentId) {
-    Query query = currentSession().createQuery("select s from SampleImpl s " + "join s.parent p " + "where p.sampleId = :id");
-    query.setLong("id", parentId);
+    Criteria criteria = currentSession().createCriteria(SampleImpl.class);
+    criteria.add(Restrictions.eq("submissionId", submissionId));
```

Comment:
Does this still exist? There's no `submissionId` column in the database, and this PR takes out the `submissionDocument`.